### PR TITLE
Refactor/types

### PR DESCRIPTION
**Pull request by KotAleksei**

### CODE REVIEW

File: src/components/accordion/useAccordion.tsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import * as React from 'react';
-import { PUITheme, ThemeMode } from 'utils/types';
 import { useCombinedRefs } from 'utils/hooks/combinedrefs';
+import { PUITheme, ThemeMode } from '../../utils/types';
```
Comment (GolotaMishka) [inline src/components/accordion/useAccordion.tsx:3]: You, can create index file inside utils where you export everything and then you can write import { useCombinedRefs, PUITheme, ThemeMode } from 'utils'; And why for one case you use relative path, for another absolute?

File: src/components/accordion/useAccordion.tsx [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 import { useAccordionContext } from './group';
 
-interface UseAccordinProps {
+export interface UseAccordinProps {
   ref?: any;
   onClick?: any;
```
Comment (GolotaMishka) [inline src/components/accordion/useAccordion.tsx:8]: onClick: (event: MouseEvent) => void; ?

File: src/components/accordion/useAccordion.tsx (hunk 1 above)
Comment (KotAleksei) [reply src/components/accordion/useAccordion.tsx:3]: for correct using types from UI in typescript project. 
after build and download package from npms in node_modules inside the package you should to use absolute paths - idk why but for types for sure

File: src/components/accordion/useAccordion.tsx (hunk 2 above)
Comment (KotAleksei) [reply src/components/accordion/useAccordion.tsx:8]: yes)

**Review by GolotaMishka — APPROVED** [review]

File: src/components/accordion/useAccordion.tsx (hunk 1 above)
Comment (KotAleksei) [reply src/components/accordion/useAccordion.tsx:3]: it's good approach but it doesn't work for ts lib as far as I know

**Pull request merged**